### DISCUSSION TITLE
Repeat & adding up numbers in a loop

### DISCUSSION THREAD

**Discussion by wirmachenbunt** [cables-gl/cables_docs#323] [Answers & Questions] [answered]
let's say i generate different numbers within a repeat region and would like to add them together. is this possible out of the box ?

something like this, 3 iterations of repeat creating random numbers and i want each iteration the sum of all previous values

0.5 = 0.5
1.2 = 1.7
0.3 = 2.0

AddUp op would do the job if it just triggers in each iteration

**Comment by wirmachenbunt** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/1138673/186117586-9e332bde-32fb-4693-ad9d-15b383449ac5.png)
maybe not really elegant but it works ...

**Reply by pandrr** [reply]
you can connect the addup trigger directly to the repeat

**Reply by wirmachenbunt** [reply]
but then it counts up indefinetely
